### DISCUSSION TITLE
I installed Ubuntu for Rock5b on Orange Pi 5 Plus and it seemed to work fine, except for the wired LAN.

### DISCUSSION THREAD

**Discussion by ironiris** [Joshua-Riek/ubuntu-rockchip#189] [General]
I prepared Ubuntu for OrangePI 5 on a microSD card and failed to boot it on OrangePI 5 Plus,
I can install Ubuntu for Rock5b on Orange PI 5 Plus and it seemed to work fine, except for the wired LAN.

To use the wired LAN, I connected a wired LAN dongle to the USB3.0 port and it was recognized.

The video coming in through HDMI IN was an SDL library problem, so I logged into xorg environment and executed the command below to check the video.

`gst-launch-1.0 v4l2src device=/dev/video0 ! video/x-raw,format=NV12,width=3840,height=2160, framerate=30/1 ! xvimagesink`

Installing to NVMe was also no problem.

**Comment by ironiris** [comment]
In OBS, the device seems to be recognized, but I can't see the video.
Is this possible? Thanks for your help.

**Reply by Joshua-Riek** [reply]
I have a new release for the Orange Pi 5 Plus uploading now, but my internet speed is very slow uploading to GitHub. Im shocked the Rock 5B image even boots on the Orange Pi 5 Plus haha. But the upload should be done in maybe 1-2 hours? All depends on the internet speed.

**Comment by Joshua-Riek** [comment]
The new release has been uploaded, and Orange Pi 5 Plus images are now available.
